src/main/java/weiss/wordle/WordleGame.java

```diff
@@ -0,0 +1,62 @@
+package weiss.wordle;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Random;
+
+public class WordleGame {
+    private final ArrayList<String> dictionary = new ArrayList<String>();
+    private final Random random = new Random();
+    CharResult correct = CharResult.Correct;
+    CharResult notPresent = CharResult.NotFound;
+    CharResult wrongPlace = CharResult.WrongPlace;
+    private String correctWord;
+
+
+    public WordleGame() throws FileNotFoundException {
+        File file = new File("src/main/java/weiss/wordle/dictionary.txt");
+        Scanner sc = new Scanner(file);
+
+        while (sc.hasNextLine()) {
+            dictionary.add(sc.nextLine());
+        }
+
+        ArrayList<String> words = new ArrayList<String>();
+        ArrayList<String> fiveLetterWords = new ArrayList<String>();
+        String[] dictLine;
+        for (int i = 0; i < dictionary.size(); i++) {
+            dictLine = dictionary.get(i).split(" ", 2);
+            words.add(dictLine[0]);
+        }
```

Comment:
You were supposed to use the WordleDictionary class you created, not copy and paste that code in here.
This is unacceptable.